### ISSUE TITLE
Add warning about server bans and kicks

### ISSUE THREAD

**Issue by ghost** [multitheftauto/mtasa-blue#3861]
### Is your feature request related to a problem? Please describe.

As an active member of the community, I and many of my teammates in the team have noticed that some server developers use similar or identical messages for bans and/or kicks, which may give users the wrong impression that they have received a global ban. Some beginners or inexperienced players are easily fooled, and there have been times when the team tried to help them, but they simply did not believe that this ban was only for the particular server they wanted to play on, as a loyal player.

As several other multiplayer modifications do, such as FiveM, it would be worthwhile for us to display a warning message in the context of the disconnect message, thus immediately informing the player that they can play on other servers.

### Describe the solution you'd like

Display a message at the bottom of the message box that clearly informs the player that the ban or kick applies to this server only and was not issued by the Multi Theft Auto team.

### Security Policy

- [x] I have read and understood the Security Policy and this issue is not about a cheat or security vulnerability.

**Comment by Fernando-A-Rocha** [comment]
This is similar to #3666

**Comment by aliemad114** [comment]
![Image](https://github.com/user-attachments/assets/8d837dc6-545c-4fe1-a221-ae7afefab095)

**Comment by aliemad114** [comment]
![Image](https://github.com/user-attachments/assets/385c9912-ec71-45ed-ba23-fb7df9388631)
What is the solution?

**Comment by aliemad114** [comment]
![Image](https://github.com/user-attachments/assets/e24ffcd3-5539-4b32-98e4-c7d3fcb209c1)
What is the solution?

**Comment by Kinimel** [comment]
I agree, some of the messages are very similar to the MTA's messages. I myself was confused when an expulsion message appeared, only to realize it was related to the server and not the MTA itself, because the server didn't allow the use of AutoHotKey, which is a program for running malicious scripts. One solution I had in mind was to rephrase the messages about bugs caused by the client's PC or the MTA component, and add an icon on the sides like 🔧 Related `Components/BUG/System`, and another one that is still being worked on to differentiate bans 🚨 `with two new icons for the MTA files.`

**Comment by Kinimel** [comment]
Here is a solution I made in another project: https://github.com/multitheftauto/mtasa-blue/issues/3666
